data/types/device/override/settings/pump/pump.go

```diff
@@ -15,7 +15,7 @@ const (
 	BasalRateScaleFactorMinimum          = 0.1
 	CarbohydrateRatioScaleFactorMaximum  = 10.0
 	CarbohydrateRatioScaleFactorMinimum  = 0.1
-	DurationMaximum                      = 604800 // 7 days in seconds
+	DurationMaximum                      = 604800000 // 7 days in milliseconds
```

Comment:
```suggestion
	DurationMaximum                      = int((7 * 24 * time.Hour).Milliseconds())
```
No comment necessary.

Reply:
fyi this is consistent with how we have expressed durations in other places in device data